tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -672,6 +676,51 @@ def convert_pytorch_to_onnx(model, input_shape, opset_version, example_inputs, o
     return model_onnx
 
 
+def convert_pytorch_pytorch_frontend(model, input_shape, example_inputs):
+    import torch
+    try:
+        from openvino.frontend import FrontEndManager
+        from openvino.frontend.pytorch.decoder import TorchScriptPythonDecoder, pt_to_ov_type_map
+        fe_manager = FrontEndManager()
+        fe = fe_manager.load_by_framework('pytorch')
+    except Exception as e:
+        print("PyTorch frontend loading failed")
+        raise e
+    inputs = prepare_torch_inputs(example_inputs, input_shape)
+    if isinstance(inputs, dict):
+        example_inputs_list = list(inputs.values())
```

Comment:
Not sure that this issue wasn't there before (in ONNX export part), but what about order of this inputs, why should it be the same as PT model expects?

Reply:
onnx support dict as example_inputs, I'm not sure that torchscript support them.
After checking I found answer that it is supported, so I'll update

Reply:
updated

Reply:
Need more information about how it woks. Are keys used as parameter names in forward method?

Reply:
Yeah, already see this in the code.